pages/index.vue

```diff
@@ -102,77 +112,30 @@
 
             stringifiedDataDictionary() {
 
-                // 0. Return a blank string if there is no loaded data dictionary file
-                if ( !this.$store.getters.isDataDictionaryLoaded ) {
-
-                    return "";
-                }
-
-                // 1. Return a string version of the data dictionary file
-                // NOTE: Defaults to json for now
-                return JSON.stringify(this.dataDictionary.original, null, 4);
+                return JSON.stringify(this.dataDictionary, null, 4);
             },
 
             stringifiedDataTable() {
 
-                // 0. Return a blank string is there is no loaded data table
-                if ( !this.$store.getters.isDataTableLoaded ) {
-
-                    return "";
-                }
-
-                // 1. Convert the tsv file data into a list of strings
-                // NOTE: Defaults to tsv for now
-                const textAreaArray = [Object.keys(this.dataTable.original[0]).join("\t")];
-                for ( let index = 0; index < Object.keys(this.dataTable.original[0]).length; index++ ) {
-
-                    textAreaArray.push(Object.values(this.dataTable.original[index]).join("\t"));
-                }
-
-                // 2. Return the tsv file data joined as one string
-                return textAreaArray.join("\n");
+                // Returns only the cell values of the table as a formatted string (no column names)
+                return this.dataTable.map(row => {
+                    return Object.values(row).join("\t");
+                }).join("\n");
             }
         },
 
         methods: {
 
             ...mapActions([
 
-                "saveDataDictionary",
-                "saveDataTable"
+                "setDataDictionary",
+                "setDataTable"
             ]),
 
             ...mapMutations([
 
-                "createColumnToCategoryMap"
-            ]),
-
-            saveDataDictionary(p_fileData) {
-
-                // Update the store with json file data
-                // NOTE: Defaults to json for now
-                this.saveDataDictionary({
-
-                    data: p_fileData.data,
-                    filename: p_fileData.filename,
-                    fileType: "json"
-                });
-            },
-
-            saveDataTable(p_fileData) {
-
-                // 1. Update the store with tsv file data
-                // NOTE: Defaults to tsv for now
-                this.saveDataTable({
-
-                    data: ( 0 === p_fileData.data.length ) ? null : p_fileData.data,
-                    filename: p_fileData.filename,
-                    fileType: "tsv"
-                });
-
-                // 2. Create a new map for linking table columns to annotation categories
-                this.createColumnToCategoryMap();
-            }
+                "initializeColumnToCategoryMap"
```

Comment:
@surchs This can now be removed since it will be called in the `setDataTable` action in the store – which you are working on.

Reply:
thanks, good catch!